crates/egui/src/widget_text.rs

```diff
@@ -685,11 +684,19 @@ impl WidgetText {
         style: &Style,
         fallback_font: FontSelection,
         default_valign: Align,
-    ) -> LayoutJob {
+    ) -> Arc<LayoutJob> {
         match self {
-            Self::RichText(text) => text.into_layout_job(style, fallback_font, default_valign),
+            Self::Text(text) => Arc::new(LayoutJob::single_section(
+                text,
+                crate::text::TextFormat::default(),
```

Comment:
This is also wrong, and should again copy https://github.com/emilk/egui/blob/b375687cbbba4c18077672b9497814ecccdfaebb/crates/egui/src/widget_text.rs#L406-L411